assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+const API_KEY = "b248c5c1ee789732d38aa9fe65ef7935";
+const form = document.getElementById("form");
+const currentWeatherContainer = document.getElementById("currentWeather");
+const forecastWeatherContainer = document.getElementById("forecastWeather");
+let city = "";
+let lat = "";
+let lon = "";
+let temp = "";
+let icon = "";
+let wind = "";
+let humidity = "";
+let description = "";
+let tempDate = "";
+let uv = "";
+const renderRecentCities = () => {
+  const container = document.getElementById("recent");
+  const searchContainer = document.getElementById("list");
+  searchContainer.remove();
+  $("#recent").append(` <div id="list" class="list-group">
+            
+  </div>`);
+  // get recent cities from LS []
+  const recentCities = JSON.parse(localStorage.getItem("recentSearches"));
+  // if [] is empty then render alert
+  if (recentCities !== null) {
+    for (i = 0; i < recentCities.length; i++) {
+      let city = recentCities[i];
+      $("#list")
+        .append(`<a  data-value="${city}" id="${i}"class="list-group-item list-group-item-action">
+     ${city}
+    </a>
+  `);
+    }
+  }
+  // else render all recent cities
+  // add an event listener on div containing all cities
+  $("#list").click(handleRecentSubmit);
+};
+
+const renderCurrentWeather = (currentWeatherData) => {
+  // render the current weather data and append to section
+  let weatherContainer = document.getElementById("weatherContainer");
+  $("#weatherContainer").css(
+    "background-image",
+    `url('./assets/images/backgrounds/${icon}.jpeg')`
+  );
+  $("#currentWeather").append(`  <div  id="currentWeather" class="border row">
+  <div  class="col-auto">
+    <h1 class="title">${city}, ${date} <img src="./assets/images/png/${icon}.png" class="img-fluid" alt="..." /></h1>
+    <h2 class="title"> ${temp}°C</h2>
+  </div>
+           
+            <div class="col-auto">
+             
+            <p class="weatherDescription">Wind: ${wind}  mph</p>
+            <p class="weatherDescription">Humidity:${humidity}%</p>
+            <p class="weatherDescription">${description}</p>
+          </div>
+           
+         
+          <div class="col-sm-12">
+            
+          </div>
+        </div>
+`);
+};
+
+const renderForecastWeather = (forecastWeatherData) => {
+  // render the forecast weather data and append each card to section
+  $("#forecastWeather").append(`  <div class="card" style="width: 10rem">
+  <img src="./assets/images/png/${icon}.png" class="card-img-top rounded float-start" alt="..." />
+  <div class="card-body">
+    <h5 class="card-title title">${date}</h5>
+    <p class="weatherDescription">Temp:${temp}°C</p>
+   </div>
+</div>`);
+};
+
+const renderWeatherData = (search) => {
+  const currentWeatherUrl = `https://api.openweathermap.org/data/2.5/weather?q=${search}&units=metric&appid=${API_KEY}`;
+  console.log(currentWeatherUrl);
+  let items = [];
```

Comment:
Arrays should generally be declared with `const`, since we don't usually reassign the value, but instead we will be adding things into the array.